src/js/player.js

```diff
@@ -287,11 +323,95 @@ class Player extends Component {
     this.tech = false;
   }
 
+  addTechControlsListeners() {
+    let userWasActive;
+
+    // Some browsers (Chrome & IE) don't trigger a click on a flash swf, but do
+    // trigger mousedown/up.
+    // http://stackoverflow.com/questions/1444562/javascript-onclick-event-over-flash-object
+    // Any touch events are set to block the mousedown event from happening
+    this.on(this.tech, 'mousedown', this.onTechClick);
+
+    // If the controls were hidden we don't want that to change without a tap event
+    // so we'll check if the controls were already showing before reporting user
+    // activity
+    this.on(this.tech, 'touchstart', function(event) {
+      userWasActive = this.userActive();
+    });
+
+    this.on(this.tech, 'touchmove', function(event) {
+      if (userWasActive){
+        this.reportUserActivity();
+      }
+    });
+
+    this.on(this.tech, 'touchend', function(event) {
+      // Stop the mouse events from also happening
+      event.preventDefault();
+    });
+
+    // Turn on component tap events
+    this.tech.emitTapEvents();
```

Comment:
We should probably just turn this on permanently in the Tech constructor. The player should already handle listening for it or not.